### DISCUSSION TITLE
Personal drive + drive_id sync

### DISCUSSION THREAD

**Discussion by erhapp** [abraunegg/onedrive#2298] [Q&A] [answered]
Hello

I'm somewhat lost configuring onedrive.

I'm unable to figure out how to sync both my "personal" folders and a sharepoint library (using the drive_id option in config)

When authenticating the system will sync my "personal" folders but after adding the drive_id to "config" only the sharepoint library is synced.

How can I have both?

kind regards

Erik

**Comment by abraunegg** [comment, marked as answer]
@erhapp 
> When authenticating the system will sync my "personal" folders but after adding the drive_id to "config" only the sharepoint library is synced.
> 
> How can I have both?

Please read:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#additional-configuration 
* https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md

Essentially you need 2 configurations - one for your 'personal' and one for your 'SharePoint Library'

**Comment by erhapp** [comment]
@abraunegg Ohh, two configs. This is exactly what I was looking for. Thank you very much.
